samples/AnomalyDetector/src/Generated/tspCodeModel.json

```diff
@@ -1060,8 +988,7 @@
           "Name": "ErrorResponse",
           "Namespace": "AnomalyDetector.Multivariate",
           "Description": "ErrorResponse contains code and message that shows the error information.",
-          "IsNullable": false,
-          "Usage": "RoundTrip",
+          "Usage": "Output",
```

Comment:
the usage changed. Why? Is the new usage correct?

Reply:
For this, I think it is. `ErrorResponse` only only in response, so it should be `output`